src/ai/chatAi.ts

```diff
@@ -0,0 +1,54 @@
+import Ain from '@ainblockchain/ain-js';
+import Ainize from '@ainize-team/ainize-js';
+
+import Ainft721Object from '../ainft721Object';
+import Assistants from './assistants/assistants';
+import { ChatConfigureParams, TransactionResult } from '../types';
+import {
+  buildSetTransactionBody,
+  validateObject,
+  validateObjectOwnership,
+  validateService,
+  validateServiceName,
+} from '../util';
+
+export default class ChatAi {
+  assistants: Assistants;
```

Comment:
(minor) maybe this definition can be moved after ain and ainize definitions..

Reply:
Done!